flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/src/FileSourceTextLinesITCase.java

```diff
@@ -84,18 +168,33 @@ public void testBoundedTextFileSource() throws Exception {
 		writeHiddenJunkFiles(testDir);
 
 		final FileSource<String> source = FileSource
-				.forRecordStreamFormat(new TextLineFormat(), Path.fromLocalFile(testDir))
-				.build();
+			.forRecordStreamFormat(new TextLineFormat(), Path.fromLocalFile(testDir))
+			.build();
 
-		final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+		final StreamExecutionEnvironment env = new TestStreamEnvironment(miniCluster, PARALLELISM);
 		env.setParallelism(PARALLELISM);
+		env.enableCheckpointing(10L);
```

Comment:
I think the bounded case should have explicitly no checkpointing enabled, so that we have recovery that re-executes the fully (goes back to the beginning).

Reply:
makes sense